src/libs/actions/PaymentMethods.js

```diff
@@ -109,12 +109,26 @@ function setWalletLinkedAccount(password, bankAccountID, fundID) {
                     walletLinkedAccountID: bankAccountID || fundID, walletLinkedAccountType: bankAccountID ? CONST.PAYMENT_METHODS.BANK_ACCOUNT : CONST.PAYMENT_METHODS.DEBIT_CARD,
                 });
                 Growl.show(Localize.translateLocal('paymentsPage.setDefaultSuccess'), CONST.GROWL.SUCCESS, 5000);
-            } else {
-                Growl.show(Localize.translateLocal('paymentsPage.setDefaultFailure'), CONST.GROWL.ERROR, 5000);
+                return;
+            }
+            Growl.show(Localize.translateLocal('paymentsPage.error.setDefaultFailure'), CONST.GROWL.ERROR, 5000);
+        }).catch((error) => {
+            // Make sure to show user more specific errors which will help support identify the problem faster.
+            switch (error.message) {
+                case CONST.WALLET.ERROR.INVALID_WALLET:
+                case CONST.WALLET.ERROR.NOT_OWNER_OF_BANK_ACCOUNT:
+                    Growl.show(Localize.translateLocal('paymentsPage.error.notOwnerOfBankAccount') + " " + Localize.translateLocal('common.conciergeHelp'), CONST.GROWL.ERROR, 5000);
```

Comment:
As per Ionatan suggestion, I have put the general `Please contact Concierge` message to separate translation so it can be reused. I have concatenated the two messages like this but I am not sure if there is a preference over doing this or using the \`${foo} ${bar}\` way.

Reply:
Alright, linter has answered this for me by throwing an error :)